CruzHacks/cruzhacks-2024-website
# Close Applications

# Scope

- Close applications

# Implementation

- Create close applications page, remove apply links.

# How to Test

- Navigate to /apply
- Check un-authed homepage for apply link
